### PR TITLE
Fix MacOS Support

### PR DESCRIPTION
**Pull request by Sploder12**
# Description

The previous change had an issue with the cross-compiler and tools.
Specifically, `i386-...` no longer exists and instead installs `x86_64-...`
This change fixes the brew commands and updates the Makefile

Issue #40 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)

### Testing 
I cannot physically test this myself.

# Checklist:
	
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,9 +2,9 @@ PLATFORM := $(shell uname)
 DEBUG ?= false # lowers optimization levels and increases command verbosity
 
 ifeq ($(PLATFORM), Darwin) 
-CC := i386-elf-gcc
-LD := i386-elf-ld
-OBJCOPY := gobjcopy
+CC := x86_64-elf-gcc
+LD := x86_64-elf-ld
+OBJCOPY := x86_64-objcopy
```
Comment (p5quared) [inline Makefile:7]: In my `/opt/homebrew/bin` I do not have `x86_64-objcopy`,  but `x86_64-elf-objcopy`, as a result from installing the binutils.

Either I am using the wrong one or this should be -elf- as well?

Comment (Sploder12) [reply Makefile]: You're right, good catch!

**Comment by emilkovacev** [comment]
PS2 functionality breaks on both systems with this change. I realize I *may* have used a different cross-compiler than the one I linked, I figured out the packages I installed that make it work properly, and will make a PR soon [tm]. I can test this on both intel and m1 mac.

**Comment by Sploder12** [comment]
> PS2 functionality breaks on both systems with this change. I realize I may have used a different cross-compiler than the one I linked, I figured out the packages I installed that make it work properly, and will make a PR soon [tm]. I can test this on both intel and m1 mac.

I have determined the cause of the PS2 breaking and pushed a theoretical fix

**Comment by p5quared** [comment]
I can say, anecdotally, this appears to work on my M1 Mac. 🙇‍♂️ 👏 🙇‍♂️ 
ezgif-5-7b1f075c24

**Review by Dylanz5 — APPROVED** [review]
yup

**Pull request merged**